<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>21</y>
     <width>78</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>QRZ Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>64</y>
     <width>102</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>QRZ Password:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>106</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>QRZ URL:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>198</y>
     <width>95</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CloudLog API:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>240</y>
     <width>100</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CloudLog URL:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>338</y>
     <width>94</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RigControl IP:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>380</y>
     <width>110</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>RigControl Port:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="useqrz_checkBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>84</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Use QRZ</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="usecloudlog_checkBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>119</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Use CloudLog</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="userigcontrol_checkBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>127</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Use RigControl</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qrzname_field">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>W1AW</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qrzpass_field">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Secret</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qrzurl_field">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>http://xmldata.qrz.com/xml/</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cloudlogapi_field">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>cl12345678901234567890</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cloudlogurl_field">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>411</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>http://www.yoururl.com/Cloudlog/index.php/api/qso</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rigcontrolip_field">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>336</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rigcontrolport_field">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>376</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>4532</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>171</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
